server/text_generation_server/layers/attention/xpu.py

```diff
@@ -0,0 +1,79 @@
+import intel_extension_for_pytorch as ipex
+import torch
+
+SUPPORTS_WINDOWING = False
+
+
+def attention(
+    q,
+    k,
+    v,
+    out,
+    cu_seqlens,
+    max_s,
+    softmax_scale,
+    window_size_left=-1,
+):
+    if window_size_left <= 0 and window_size_left != -1:
+        raise ValueError("`window_size_left` must be > 0 or -1")
+
```

Comment:
I guess this condition can be removed as well, given that window attention is not supported.

Reply:
But below an exception is raised `window_size_left != -1`, so all window sizes other than `-1` are impossible.